README.md

```diff
@@ -18,40 +18,74 @@ Some of these are tricky!  Don't trust your first instinct.
 
 **a)** `"1" == 1`
 
+True, this is because it is comparing the values so even though one is a string it is coercing the value to be true.
+
 **b)** `"1" === 1`
 
+False, because when using triple equals you are comparing the pure values and a string does not equal the number.
+
 **c)** `x == 'x'`
 
+When x is undefined it cannot equal the other
+
 **d)** `x == (x+'')`
 
 **e)** `'' == ' '`
 
+False because the quotes do not equal the same type.
+
 **f)** `x = true`
 
+True, because we are defining x and it is a value therefore true.
+
 **g)** `var x; x == 'undefined'`
 
+False, because undefined is considered not a number therefore it cannot be equal to not a number because there is no value.
+
 **h)** `'9'<'10'`
 
+ True, because we have the less than symbol and 9 is less than 10
+
 **i)** `typeof x + 1 === "number"`
 
+  False because x + 1 does not equal an numbered string
+
 **j)** `typeof x % 2 === "number"`
 
+False because again the numbered string is not purely equal to the x we are using that is undefined.
+
 **k)** `typeof (x % 2) === "number"`
 
+False because again the typeof is not a string therefore it cannot equal a string
+
 **l)** `x++ == ++x`
 
+False because the type is not the same
+
 **m)** `++x == x++`
 
+  True, because both types are similar rather than undefined.
+
 **n)** `"1"+x == 1+x`
 
+ False, because adding a string to x is not the same as adding the number 1 to x
+
 **o)** `"0"+1 == 1`
 
+ True, because string 0 is undefined which means its 1 is equal to 1 both numbers.
+
```

Comment:
In this case 0 is not undefined, by the very nature of it being 0. You are mostly only going to see variables as undefined.